en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-200-to-300.md

```diff
@@ -412,16 +412,11 @@ Follow the instructions below to move all the existing API Manager configuration
     !!! note
         It is recommended to use the default H2 database for the `WSO2_MB_STORE_DB` database in API-Manager. So do **not** migrate `WSO2_MB_STORE_DB` database from API-M 2.0.0 version to API-M 3.0.0 version, and use the **default H2** `WSO2_MB_STORE_DB` database available in API-M 3.0.0 version.
 
-4.  Update `<API-M_3.0.0_HOME>/repository/resources/conf/default.json` file by pointing to the WSO2UM_DB.
+4.  Update `<API-M_3.0.0_HOME>/repository/conf/deployment.toml` file as follows to update the datasource name under the realm manager configurations in the `user-mgt.xml` file by pointing to the WSO2UM_DB.
```

Comment:
``` suggestion
4.  Update `<API-M_3.0.0_HOME>/repository/conf/deployment.toml` file as follows, to point the correct database for user management purpose
```
We do not need to mention regarding user-mgt.xml. Shall we remove it?

Reply:
+1. Will modify.